data/json/items/comestibles/mutagen.json

```diff
@@ -460,302 +464,295 @@
     "id": "mutagen_alpha",
     "copy-from": "mutagen_flavor",
     "type": "COMESTIBLE",
-    "name": { "str_sp": "alpha mutagen" },
-    "description": "An extremely rare mutagen cocktail.",
+    "name": { "str_sp": "alpha mutagenic slurry" },
     "price": "5000 USD",
     "looks_like": "mutagen",
     "//": "*May* have gotten a dose smuggled out.",
     "use_action": {
       "type": "consume_drug",
-      "activation_message": "You drink the alpha mutagen.",
-      "vitamins": [ [ "mutagen_alpha", 225 ], [ "mutagen", 125 ] ]
+      "activation_message": "You drink the alpha mutagenic slurry.",
+      "vitamins": [ [ "mutagen_alpha", 225 ], [ "mutagenic_slurry", 15 ] ]
```

Comment:
```suggestion
      "vitamins": [ [ "mutagen_alpha", 225 ], [ "mutagenic_slurry", 125 ] ] did you intend to go from 125 to 15 mutagenic_slurry vitamins?  It looks like most of them have this error if it is an error. 
```

Reply:
Intentional. 1 `mutagenic_slurry` decays into 8 `mutagen` (among other things), so in terms of that it's close (125 vs 120).